docs/_data/2017.eu.speakers.yaml

```diff
@@ -509,7 +509,7 @@
     title: 'Tech Writers Without Borders: Making the world a better place, one (numbered) step at a time'
     slides: https://speakerdeck.com/sculshaw/making-the-world-a-better-place-one-numbered-step-at-a-time
     event: Write the Docs EU 2017
-    path: conf/eu/2017/videos/
+    path: conf/eu/2017/videos/tech-writers-without-borders-making-the-world-a-better-place-one-numbered-step-at-a-time-stuart-culshaw
```

Comment:
For some reason this isn't working

Reply:
His link seems still to be broken https://deploy-preview-828--dentist-canary-64055.netlify.com/videos/eu/2017/